libsolc/libsolc.h

```diff
@@ -42,9 +43,11 @@ extern "C" {
 /// @param o_contents A pointer to the contents of the file, if found.
```

Comment:
`A pointer to the contents of the file, if found, allocated via solidity_allocate()`.